src/graphql/types/scalar/wallet-id.ts

```diff
@@ -0,0 +1,27 @@
+import { checkedToWalletPublicId } from "@domain/wallets"
+import { GT } from "@graphql/index"
+import { UserInputError } from "apollo-server-errors"
+
+const WalletId = new GT.Scalar({
+  name: "WalletId",
+  description: "Unique identifier of a user",
+  parseValue(value) {
+    return validWalletIdValue(value)
+  },
+  parseLiteral(ast) {
+    if (ast.kind === GT.Kind.STRING) {
+      return validWalletIdValue(ast.value)
+    }
+    return new UserInputError("Invalid type for WalletId")
+  },
+})
+
+function validWalletIdValue(value) {
+  const checkedWalletId = checkedToWalletPublicId(value)
+  if (checkedWalletId instanceof Error) {
+    return checkedWalletId
+  }
+  return new UserInputError("Invalid value for WalletId")
```

Comment:
It looks like this returns an error here on the happy path

Reply:
Pushed a fix to this. Please verify.

Reply:
Confirmed, works 👍🏾 

---

```gql
mutation lnInvoiceCreateOnBehalfOfRecipient($input: LnInvoiceCreateOnBehalfOfRecipientInput!) {
  lnInvoiceCreateOnBehalfOfRecipient(input: $input) {
    errors {
      message
    }
    invoice {
      paymentRequest
      paymentHash
      paymentSecret
			satoshis
    }
  }
}

{"input": {"recipientWalletId": "60b8b880-f7ba-41e9-a4de-ea7cca19912b", "amount": "100", "memo": "This is a test"}}

```